pyartcd/pyartcd/pipelines/rebuild.py

```diff
@@ -603,9 +606,12 @@ async def rebuild(runtime: Runtime, ocp_build_data_url: str, version: str, assem
             runtime.logger.warning('Env var BUILD_URL has not been defined: '
                                    'a random identifier will be used for the locks')
 
-        await locks.run_with_lock(
-            coro=pipeline.run(),
-            lock=lock,
-            lock_name=lock_name,
-            lock_id=lock_identifier
-        )
+        await asyncio.gather(*[
```

Comment:
idk if it's worth it to have these changes in rebuild job right now.. we very rarely use rebuild job. If our scheduled plashet sync in ocp4 which runs regularly has these changes in - i.e. running for both hosts, then i think that is sufficient for testing and confirmation that everything works. Since we will be reverting back to running only on one host, so I'm thinking maybe not touch the rebuild job or very minimal changes like constant names etc

Reply:
`rebuild` hasn't run in a long time, but I was told this job was meant to be used with custom hotfixes. So it was suggested not to deprecate it for the moment.

This means `rebuild` also has to deal with a plurality of `PLASHET_REMOTES`. We could indeed get the first element in the list, but as you pointed out in a previous comment this could be cause of confusion with respect to which host we're rebuilding for.

In my opinion, `rebuild` can definitely handle multiple rebuilds in parallel; we're making use of asyncio tasks to optimize time usage, besides we'll rarely run this pipeline if ever. I would leave it like this.